utils/mock-data-loader/bss/facilities/deal-9-facilities.js

```diff
@@ -68,7 +68,7 @@ module.exports = [
     'requestedCoverStartDate-year': twoMonths.year,
     bondBeneficiary: '',
     ukefExposure: '360,000.00',
-    lastEdited: nowTimestamp,
+    updatedAt: nowTimestamp,
```

Comment:
is this `nowTimestamp` being built using date-fns? or `Date.now()` and does it matter here which one do we use?
(this is for the other files listed above)

Reply:
Yes it's using `Date.now()` from date-fns, just renamed it here